app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/non_calculated_summary.scala.html

```diff
@@ -123,6 +123,19 @@
         Some(cyaChange(messages(s"nonCalculatedTaxDue$furtherReturnKey.cyaChange"), routes.YearToDateLiabilityController.nonCalculatedEnterTaxDue().url)),
         "nonCalculatedTaxDue-value"
     )
+
+    @if(isFurtherReturn.contains(true)) {
+        @answers.checkForRepayment.map { checkForRepayment =>
+            @cyaRow(
+                messages(s"repayment$userKeyWithSeparatePeriodOfAdmin.title"),
```

Comment:
Do we need some {} here to delimit the variable name?

Reply:
Oh nevermind. It's one big name, not userKey and something else

Reply:
it works it out ok without the braces - the variable name automatically stops at the `.` - it's just a long name